docs/CreateEdit.md

```diff
@@ -317,11 +317,11 @@ const number = value => value && isNaN(Number(value)) ? 'Must be a number' : und
 const minValue = min => value =>
   value && value < min ? `Must be at least ${min}` : undefined;
 
-const ageValidation = (value, values) => {
+const ageValidation = (value, allValues) => {
     if (!value) {
         return 'The age is required';
     }
-    if (age < 18) {
+    if (allValues.age < 18) {
```

Comment:
I meant `allValues.age` should simply be `value`.

Reply:
I actually thought allValues.age would demonstrate how to use the allValues parameter.  But I agree with your opinion too.  I updated the commit.  Thanks!